webui/react/public/index.html

```diff
@@ -6,7 +6,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Determined Deep Learning Training Platform" />
-  <link rel="icon" href="%PUBLIC_URL%/favicons/favicon.ico" />
+  <link rel="shortcut icon" href="%PUBLIC_URL%/favicons/favicon.ico" />
```

Comment:
Looking back, for web standards, it might be best to keep `favicon.ico` as the base of the web root, since a lot of browsers look for it there.
```
<link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" type="image/x-icon" />
```

Reply:
`public/manifest.json` has a reference to `favicons/favicon.ico` also that needs to be changed to `favicon.ico`